java/src/jmri/jmrit/operations/rollingstock/cars/CarsTableModel.java

```diff
@@ -259,6 +259,7 @@ synchronized void updateList() {
         return getCarList(_sort);
     }
 
+    @edu.umd.cs.findbugs.annotations.SuppressFBWarnings(value = "DB_DUPLICATE_SWITCH_CLAUSES")
```

Comment:
@DanielBoudreau Please add a `justification="[something]"` to these SuppressFBWarnings as per http://jmri.org/help/en/html/doc/Technical/FindBugs.shtml as those without a justification are liable to be removed in the future.
Thanks.